internal/lightcone/abundance/postopconversation/postopconversation.go

```diff
@@ -0,0 +1,67 @@
+package postopconversation
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	PostOpErrNCheck key.Modifier = "post-op-conversation-err-and-check"
+	PostOpHeal 		key.Modifier = "post-op-conversation-heal-buff"
+)
+
+func init() {
+	lightcone.Register(key.PostOpConversation, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_ABUNDANCE,
+		Promotions:    promotions,
+	})
+
+	//Implement checker here
+	modifier.Register(PostOpErrNCheck, modifier.Config{
+		Listeners: modifier.Listeners{
+			//NOTE : DM uses OnBeforeDealHeal instead of OnBeforeAction. Might need change.
+			OnBeforeAction: buffHealsOnUlt, 
+			OnAfterAction: removeHealBuff,
+		},
+	})
+	//The actual buff modifier goes here
+	modifier.Register(PostOpHeal, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
+	})
```

Comment:
We do not want this registered as a buff because the game does not (if we register this as a buff, will cause other logic to behave different from game like buff count checks).